### PR TITLE
Image Viewer Camera

### PR DESCRIPTION
**Pull request by mazunin-v-jb**
Related issue: https://github.com/JetBrains/compose-multiplatform/issues/2774

Also, this PR fixes crash https://github.com/JetBrains/compose-multiplatform/issues/2845

### CODE REVIEW

**Review by dima-avdeev-jb — APPROVED** [review]

File: experimental/examples/imageviewer/gradle.properties [hunk 1]
```diff
@@ -13,5 +13,5 @@ kotlin.native.useEmbeddableCompilerJar=true
 kotlin.native.binary.memoryModel=experimental
 kotlin.version=1.8.0
 agp.version=7.1.3
-compose.version=1.4.0-alpha01-dev972
+compose.version=1.4.0-alpha01-dev975
```
Comment (dima-avdeev-jb) [inline experimental/examples/imageviewer/gradle.properties:16]: This version upgrade fixes crash https://github.com/JetBrains/compose-multiplatform/issues/2845

**Review by dima-avdeev-jb — APPROVED** [review]

File: experimental/examples/imageviewer/shared/src/iosMain/kotlin/example/imageviewer/view/CameraView.ios.kt [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+            }
+        }
+    }
+    Box(
+        Modifier.fillMaxSize().background(Color.Black),
+        contentAlignment = Alignment.Center
+    ) {
+        when (cameraAccess) {
+            CameraAccess.Undefined -> {
+                Text("Camera needs user permission", color = Color.White)
```
Comment (pjBooms) [inline experimental/examples/imageviewer/shared/src/iosMain/kotlin/example/imageviewer/view/CameraView.ios.kt:63]: an user permission

File: experimental/examples/imageviewer/shared/src/androidMain/kotlin/example/imageviewer/view/CameraView.android.kt [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+
+@Composable
+internal actual fun CameraView(modifier: Modifier) {
+    Box(Modifier.fillMaxSize().background(Color.Black)) {
+        Text(
+            text = "For now CameraView is not working on Android",
```
Comment (pjBooms) [inline experimental/examples/imageviewer/shared/src/androidMain/kotlin/example/imageviewer/view/CameraView.android.kt:16]: Camera is not available on Android for now.

File: experimental/examples/imageviewer/shared/src/desktopMain/kotlin/example/imageviewer/view/CameraView.desktop.kt [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+
+@Composable
+internal actual fun CameraView(modifier: Modifier) {
+    Box(Modifier.fillMaxSize().background(Color.Black)) {
+        Text(
+            text = "For now CameraView is not working on Desktop",
```
Comment (pjBooms) [inline experimental/examples/imageviewer/shared/src/desktopMain/kotlin/example/imageviewer/view/CameraView.desktop.kt:16]: Camera is not available on Desktop for now

**Review by pjBooms — APPROVED** [review]

**Pull request merged**